STA/azure/sta_ng.tf

```diff
@@ -8,19 +8,25 @@ variable "Prefix" {}
 variable "DiskSize" {}
 variable "Region" {}
 variable "STA_Version" {}
+variable "storageAccountResourceGroup" {}
+variable "storageAccount" {}
+variable "storageContainer" {}
 
 
 module "STA" {
```

Comment:
test your work!

Reply:
tested and working (had to change the image source in Module's Resources file to get my PR image).
after adding the new keys, the STA is syncing successfully with the remote storage at initialization phase